### ISSUE TITLE
Calculation for LIC kit to be changed to 5k + words changed 

### ISSUE THREAD

**Issue by Payoshni** [PrathamBooks/frp#263]
as discussed with @kuchlous , but changes to be implemented only after August 2nd.

On the Beneficiary Form at Step 2: https://donateabook.org.in/setup

'What are you raising funds for':
For Books - calculation to be same: 1 x INR 36.75
For LIC Calculation to be changed.Change from 125 books x 36.75 to  INR 5000 flat

So, if I am selecting 2 LIC kits, the system should show 10k as funding amount.
If i am selecting 2 LIC kits and 200 books, the system should show 10k + (200x 36.75) 7350 = 17350 as funding amount

Also changes to be made to the text under the box: 

Library-in-a-classroom (LIC)
Note - Each LIC costs Rs.5000 and contains approximately 125 books.

![image](https://cloud.githubusercontent.com/assets/12147445/8954440/412b114e-3603-11e5-8537-e57ea409e440.png)

**Comment by kuchlous** [comment]
As discussed this will mean that the number we show for "Books Pledged So
Far" will become approximate (we will not count the LIC books separately)
and continue to use the total money raised divided by 36.75 to arrive at
this number.

**Comment by Payoshni** [comment]
Yes, okay as discussed. Post 2nd August then. thanks

**Comment by Payoshni** [comment]
@kuchlous  please do update this is not already done.

**Comment by kuchlous** [comment]
@Payoshni , any updates on this? If this is not needed, I will remove the code.

**Comment by Payoshni** [comment]
@kuchlous No decision yet. Suzie is expected on Friday so post that. Can you just keep it on hold till then.

**Comment by kuchlous** [comment]
Superseded by #276 .
